bip-0018.mediawiki

```diff
@@ -125,7 +125,7 @@ This BIP only allows Bitcoin's Hash160 algorithm, but leaves open the possibilit
 
 ==Reference Implementation==
 
-https://github.com/gavinandresen/bitcoin-git/tree/pay_to_script_hash
+https://github.com/bitcoin/bips/blob/master/bip-0016.mediawiki
```

Comment:
Pinging BIP author @luke-jr for feedback on this change.

Reply:
This is another BIP, not an implementation...